src/locales/en.json

```
@@ -110,38 +110,38 @@
     "issue-button": "Suggest a new source",
     "aria": { "table": "sources table" }
   },
-  "meta-search-page": {
-    "title": "Meta Search",
+  "additional-sources-page": {
+    "title": "Additional Sources",
     "intro": "{openverse} is built on top of a catalog that indexes CC-licensed and public domain content from selected sources. Learn more about our {link}.",
     "link": "sources",
     "license": "The internet is a big place! There are plenty of search portals and collections, hosting CC-licensed and public domain content, which we aren’t able to index or surface results from within the {openverse} interface. These are important sources of CC-licensed and public domain content, and we want to make sure that you are able to find the best openly licensed materials possible, regardless of where they are located.",
-    "content": "What content types can I search for using the Meta Search feature?",
-    "content-types": "Meta Search on {openverse} currently supports image, audio, and video search on external sites.",
+    "content": "What content types can I search for using the Additional Sources feature?",
```

Example of a grammatical error introduced because "Additional Sources" is not the proper-noun of a feature name but rather an adjective-qualified common-noun.
```suggestion
    "content": "What content types are supported by additional sources?",
```

---

Just a nit here, but this isn't technically a grammatical error, and in fact could maybe even be clearer than what you suggested in context. At least in this case the phrase 'Additional Sources feature' groups it as a single block. In other strings you're right that it reads awkwardly to just replace "Meta Search" directly with "Additional Sources", but only because it's not qualifying "feature".

In general I think the wording is probably bad here because we're treating "Additional Sources" (even if lower cased) as a feature and also the name of a collection of external resources. I think it could be a good idea to revisit this group of strings altogether instead of doing a 1-to-1 term replacement. For example, this particular string is really confusing to read as it could be asking either "what content types do the websites listed in the additional sources feature support" or it could read "what content types does Openverse integrate 'additional sources' into"... both of which expose what I think is still a really unclear name we've chosen in "additional sources" (it's maybe even more unclear than the very obscure "meta search" if only because "meta search" is almost semantically meaningless the way we used it but "additional sources" is rife with ways of misunderstanding/misinterpreting).